internal/peer/chaincode/common_test.go

```diff
@@ -159,12 +159,13 @@ func TestGetOrdererEndpointFromConfigTx(t *testing.T) {
 
 	mockchain := "mockchain"
 	factory.InitFactories(nil)
-	config := genesisconfig.Load(genesisconfig.SampleInsecureSoloProfile, configtest.GetDevConfigDir())
-	pgen := encoder.New(config)
-	genesisBlock := pgen.GenesisBlockForChannel(mockchain)
+	profile := genesisconfig.Load(genesisconfig.SampleInsecureSoloProfile, configtest.GetDevConfigDir())
+	channelGroup, err := encoder.NewChannelGroup(profile)
+	require.NoError(t, err)
+	channelConfig := &cb.Config{ChannelGroup: channelGroup}
 
 	mockResponse := &pb.ProposalResponse{
-		Response:    &pb.Response{Status: 200, Payload: protoutil.MarshalOrPanic(genesisBlock)},
+		Response:    &pb.Response{Status: 200, Payload: protoutil.MarshalOrPanic(channelConfig)},
```

Comment:
The final line of the test is:
`require.Equal(t, len(ordererEndpoints), 0)`

Is there a test that actually returns one or more orderer endpoints from the returned config, to be sure the client can still retrieve them?

Reply:
Not sure why but this test is in a different package from the source and yes, does not seem to do much. I have moved this to the right package and enhanced the test (and during this, I hit a bug related to suppression of error in the existing code).